lib/templates.js

```diff
@@ -213,14 +226,13 @@ const createEventResourceTemplate = (serviceName, eventDefinition, appConfig, pa
         extensible: { supported: "no" },
 
         ...ordExtensions
-    };
+    }]
 };
 
 function _getPackageID(namespace, packageIds, resourceType) {
-    //TODO: without this check, it will be failed in ordPackageJson.test.js, need to check why it is failing
-    if (!(packageIds instanceof Set)) return;
-    const packageIdsArray = Array.from(packageIds);
-    return packageIdsArray.find((pkg) => pkg.includes("-" + resourceType)) || packageIdsArray.find((pkg) => pkg.includes(namespace));
+    if (!packageIds) return;
+
+    return packageIds.find((id) => id.includes("-" + resourceType)) || packageIds.find((id) => id.includes(namespace));
```

Comment:
@zongqichen : As explained above, no need to check if the `packageIds` is of type `Set` because it can be an `Array` or `undefined`.